src/services/ApiNodeService.ts

```diff
@@ -118,6 +120,7 @@ export class ApiNodeService {
 		} catch (e) {
 			logger.error(`Fail to request host node status: ${host}`, e);
 			return {
+				restGatewayUrl: `http://${host}:3000`,
```

Comment:
I'm not sure about this default value.

It could happen that the rest url is on 3001 https but rest fails to get the current status. This method may swap from 3001 to 3000 when 3000 might not be there

Reply:
So if the node is not healthy or unreached, it nodes filter won't return it.
We can remove or change something `unreached` under the `restGatewayUrl`

Reply:
if the catch is caught, why still return the json?

Reply:
I just a record need to be stored in the database.